acceptance/tests/resource/ssh_authorized_key/query.rb

```diff
@@ -18,7 +18,7 @@
   on(agent, "cp #{auth_keys} /tmp/auth_keys", :acceptable_exit_codes => [0,1])
 
   step "(setup) create an authorized key in the #{auth_keys} file"
-  on(agent, "echo 'ssh-rsa mykey #{name}' >> #{auth_keys}")
+  on(agent, "echo '' >> #{auth_keys} && echo 'ssh-rsa mykey #{name}' >> #{auth_keys}")
```

Comment:
Actually, I think there's an additional issue with the error message.

```
fail_test "found the ssh_authorized_key for #{name}" unless stdout.include? "#{name}"
```

This should probably say "Couldn't find the ssh_authorized_key for #{name}". Let me check this again - I'm not sure if it really should have been passing in CentOS 4.